web/modules/custom/mymodule/mymodule.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'My Module'
+description: 'This is the first custom module.'
+type: module
+core_version_requirement: ^10
+dependencies:
```

Comment:
Are these dependencies required?

Reply:
For the sake of exploring , I have added them.

Reply:
We refrain from including unnecessary code in the repository.

Reply:
Ok. Removing it.